### PR TITLE
fix #314: --version not showing lab version

### LINKED ISSUE #314: `lab --version` does not display lab's version
I checked this on both Ubuntu1804(WSL) and Windows 10.

```shell
$ lab --version
git version 2.17.1
$ lab version # this works properly
git version 2.17.1
lab version 0.16.0
```

### PR DESCRIPTION
**Pull request by zaquestion**

### CODE REVIEW

File: cmd/project_create.go [hunk 1]
```diff
@@ -66,6 +67,7 @@ lab project create -n "new proj"           # user/new-proj named "new proj"`,
 		if err != nil {
 			log.Fatal(err)
 		}
+		spew.Dump(p)
```
Comment (claytonrcarter) [inline cmd/project_create.go]: @zaquestion did you mean to leave this in? My Go is rusty, but this looks like a debugging statement.

**Comment by claytonrcarter** [comment]
Aside from the [possible] debugging statement, the only other comment I had is that changing the default visibility from "internal" to "private" -- though needed -- seemed beyond the scope of a PR that's supposed to be fixing `--version` and I just want to confirm that you intended it to be in here. Sorry if I'm being nitpicky.

**Comment by zaquestion** [comment]
All in an effort to fix the tests, I can split it out, I'm just glad to see it seemed to work

**Comment by claytonrcarter** [comment]
All good! Just double checking that you meant it all to be there.

**Pull request merged**